mqtt 3/1.0.0/modules/ROOT/pages/edits/mqtt3-connector-quality-of-service.adoc

```diff
@@ -0,0 +1,25 @@
+= Configure the Quality of Service for MQTT 3 Connector
+
+The MQTT protocol enables you to specify levels of delivery assurance for inbound or outbound messages. These levels
+are referred to as quality of service (QoS).
+
+The available QoS levels to configure for MQTT 3 Connector are:
```

Comment:
```suggestion
are referred to as _quality of service_ (QoS).

The available QoS levels to configure for MQTT 3 Connector are:
```

Reply:
^ could be omitted altogether if you decide to use the suggested intro at Line 3.